CHANGELOG.md

```diff
@@ -5,13 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# Legend
+[C] Cere Runtime <br/>
+[D] Cere Dev Runtime
+
 ## [vNext]
 
 ### Added
 
-- Handlebars template to generate weights file
-- Genesis config for `pallet-ddc-staking` to set genesis DDC participants (empty by default) and staking settings
-- Unit tests in `pallet-ddc-staking` for basic staking scenario
+- [D] Handlebars template to generate weights file
+- [D] Genesis config for `pallet-ddc-staking` to set genesis DDC participants (empty by default) and staking settings
+- [D] Unit tests in `pallet-ddc-staking` for basic staking scenario
+
+### Changed
+
+- [C,D] Updated Substrate to polkadot-v0.9.30
```

Comment:
Will we mention each feature twice? For the first time when we release it to `[D]`, and once again when we release it to `[C]`?

Reply:
Correct